site/docs/v1/tech/installation-guide/docker.adoc

```diff
@@ -178,6 +178,19 @@ For example if startup is failing and you see the following in the logs, you wil
 2018-11-22T13:00:05.346606495Z [2]: max virtual memory areas vm.max_map_count [65530] is too low, increase to at least [262144]
 ----
 
+== Upgrading
```

Comment:
We also have an upgrading section, not sure if it belongs there or here? 
https://fusionauth.io/docs/v1/tech/installation-guide/upgrade

Reply:
Some of the install guides have upgrade sections. The upgrade page is really for the "Debian or RPM packages on Linux, or the ZIP packages on macOS, Linux or Windows."

I'll add a note to the readme on the upgrade page.